README.md

```diff
@@ -93,12 +93,19 @@ To run stress testing against the Salesforce instance, refer to the documentatio
 To get started working with our Phrase translations, you will need to:
 
 1. Install the Phrase CLI with `brew tap phrase/brewed && brew install phrase`
-1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in e.g. LastPass
+1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in Lastpass and as a local env var.
 
-**To upload new strings or translations to Phrase**
+- Each time you add new strings to locale-en.json, push to Phrase to keep it up-to-date
+- If you delete keys from locale-en.json, run phrase push, then run phrase cleanup to delete unused keys
+```
+phrase uploads cleanup --id=[upload id from the response of phrase push]  --access_token=[your access token]
```

Comment:
Can we make this part of the `phrasePush` script? As long as there's no downside to running cleanup when nothing is deleted.

Reply:
I'm not sure how to get the upload id from the output, but I'll spend a bit of time looking into it

Reply:
looks like it's not supported with grunt-exec unfortunately https://github.com/jharding/grunt-exec/pull/31. I'm going to open an issue in the phrase_cli repo to ask for a option on push to delete unused keys instead if that's alright with you!

Other option would be to spin our a separate shell file, but this seems like something other folks would want to do whenever they push too?

Reply:
@akegan yep seems like something others would want, I'm fine with just opening an issue and following up if they resolve it.

Reply:
Ok cool. Here's the issue I created: https://github.com/phrase/phrase-cli/issues/22